gtsfm/configs/colmap_front_end.yaml

```diff
@@ -1,94 +1,96 @@
 # Front-end configuration that leverages PyColmap for feature extraction and verification.
 
 # @package _global_
+defaults:
```

Comment:
Ah, no, this is different from the loader. Please stick to that pattern, and make sure partitioner ends up in right spot, i.e. after image_pairs_generator, like so:
```
loader:
  _target_: gtsfm.graph_partitioner.Single
```